alerts.go

```diff
@@ -163,6 +163,18 @@ func formatJoinedAlert(alertSet *alertSet, colorize bool) string {
 			default:
 				monitorMsg = fmt.Sprintf("%s %.2f > %.2f msec, status:%s", m.Name, alert.Value, m.ResponseTimeCritical, alert.Message)
 			}
+		case *mkr.MonitorExpression:
+			expression := trimExpression(m.Expression)
+			switch alert.Status {
+			case "CRITICAL":
+				monitorMsg = fmt.Sprintf("%s %.2f %s %.2f", expression, alert.Value, m.Operator, m.Critical)
```

Comment:
How about reporting monitor name (`m.Name`) too? I think it's useful especially when the expression is complex.